readthedocs/core/views/__init__.py

```diff
@@ -77,7 +77,9 @@ def wipe_version(request, project_slug, version_slug):
         project__slug=project_slug,
         slug=version_slug,
     )
-    if request.user not in version.project.users.all():
+    # We need to check by ``for_admin_user`` here to allow members of the
+    # ``Admin`` team (which doesn't own the project) under the corporate site.
+    if version.project not in Project.objects.for_admin_user(user=request.user):
```

Comment:
I'm changing the logic a bit here since we were checking for "the user belongs to a group" and now I'm checking if "project is under the project which the has admin permissions". Seems to be correct anyway, but just want to mentioned this change on the logic here.